deploy/charts/external-secrets/templates/webhook-certificate.yaml

```diff
@@ -13,13 +23,15 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  commonName: {{ include "external-secrets.fullname" . }}-webhook
+  commonName: "{{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}.svc"
   dnsNames:
-    - {{ include "external-secrets.fullname" . }}-webhook
-    - {{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}
-    - {{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}.svc
```

Comment:
All these are valid DNS zones for kubernetes services. By removing these, this change is a breaking one

Reply:
Breaking for what? The endpoint is a webhook, and must be reachable from the api-server. The problem with all these DNSNames, is that they are not needed. And in our setup they will break our cert-manager policies - at least the first one. But it is a change - I agree to that. Will revert if you think it's important to add all routeable DNS entries.

Reply:
@moolen do you believe this change would be acceptable? IMO it should not unless for some scenarios where people manually check webhook liveness (IDK why they would do it 🤷)

Reply:
I think for some cases it can be safe to remove this. However, we shouldn't make assumptions on how people uses this. A case could be, for instance, if someone runs a monitoring agent checks the TLS cert validity by opening a TLS connection to `{eso}-webhook.{namespace}` and verifying its legitimacy. If we remove the dnsNames that will break that user.

With that being said i prefer keeping all the `dnsNames`.